src/components/App.js

```diff
@@ -36,7 +36,7 @@ const App = () => {
           />
 
           <Route exact path="/creatures/:Creature/details" render={(detailProps) => {
-            console.log('details routeProps', detailProps);
+            // console.log('details routeProps', detailProps);
             return (
               < CreatureDetail creatureDetail={detailProps.match.params.Creature} />
             )
```

Comment:
I really like how modular your components are and how cleanly this reads. Interesting that error flashes when you use the "*" path, that was not my experience, but I like your solution of having an error route.